protos/controller.proto

```diff
@@ -5,7 +5,7 @@ import "common.proto";
 option go_package = ".;pb";
 
 // Human-readable parameters for our GJSM model.
```

Comment:
can you move this comment on top of the `Params` struct? then the swagger docs will show it